docs/getting-started/quick-start-guides/deploy-rancher-manager/helm-cli.md

```diff
@@ -117,13 +117,13 @@ helm repo update
 helm install cert-manager jetstack/cert-manager \
   --namespace cert-manager \
   --create-namespace \
-  --version v1.11.0
+  --version v1.13.1
 
 # Windows Powershell
 helm install cert-manager jetstack/cert-manager `
   --namespace cert-manager `
   --create-namespace `
-  --version v1.11.0
+  --version v1.13.1
```

Comment:
```suggestion
```

Need to drop the <code>`</code> from <code>--create-namespace &grave;</code> too.